assets/scss/frontend/_llms-quizzes.scss

```diff
@@ -165,6 +165,9 @@
 			margin: 0;
 			padding: 0;
 			position: relative;
+			label {
+				display: block;
```

Comment:
just for a matter of "cleanliness" I would rather place this `display:block` in the already existent ruleset for the label:
https://github.com/gocodebox/lifterlms/blob/3c30e10802a0c3e17c5ab2b39ce1938d45be8d01/assets/scss/frontend/_llms-quizzes.scss#L211

Reply:
Fixed!